diamm/helpers/object_pagination.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict
+import re
+
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
+
+class Object_Pagination(PageNumberPagination):
```

Comment:
Oh! I didn't notice DRF had templates. I just wanted to use the template you were using with the solr_paginator so I put together a pagination model that looked the same